psi4/driver/procrouting/proc.py

```diff
@@ -1246,7 +1246,7 @@ def scf_helper(name, post_scf=True, **kwargs):
     # Grab a few kwargs
     use_c1 = kwargs.get('use_c1', False)
     scf_molecule = kwargs.get('molecule', core.get_active_molecule())
-    read_orbitals = core.get_option('SCF', 'GUESS') is "READ"
+    read_orbitals = core.get_option('SCF', 'GUESS') == "READ"
```

Comment:
I have to admit I had to read up (again) on the difference between these two operators. You learn something everyday!